charts/moco/templates/generated/generated.yaml

```diff
@@ -373,6 +373,50 @@ spec:
     app.kubernetes.io/name: '{{ include "moco.name" . }}'
 ---
 apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: '{{ .Release.Service }}'
+    app.kubernetes.io/name: '{{ include "moco.name" . }}'
+    app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
+    helm.sh/chart: '{{ include "moco.chart" . }}'
+  name: moco-delete-validator
+  namespace: '{{ .Release.Namespace }}'
```

Comment:
Isn't this a cluster resource?

Reply:
The version of the API that kustomize is referencing is an older version.
https://github.com/kubernetes-sigs/kustomize/blob/bb7a28070905adae77c6f82b912a862de2b3a052/kyaml/openapi/Makefile#L9

Reply:
"If kustomize gets updated, we’ll be able to remove the namespace. However, until then, the CI's 'make check-generate' will fail, so we’ll keep the namespace as is. Even with the namespace attached, it will still be treated as a cluster resource, so there’s no issue."